src/pages/Home page/Home.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+import "./Home.css"
+
+function Home() {
+  return (
+    <div> <section className="hero">
+      <img alt="please reload" className="hero-image" 
+        src="https://www.decathlon.in/_next/image?url=https%3A%2F%2Fcontents.mediadecathlon.com%2Fs896860%2Fk%24382f8943451fc70c7f2ae3d4ab3db60a%2Fdesktop.jpg%3Ff%3D0x250%3Ff%3D1920x470%26format%3Dauto&w=1920&q=75" />
+    </section>
+
+      <section className="category-nav">
+        <section className="category">
+          <NavLink className="category-link" to="">
+            <img className="category-img" src="Images/jacket.jpg" alt="" />
+            <span className="category-link-text">Clothing</span>
+          </NavLink>
+        </section>
+        <section className="category">
+          <NavLink className="category-link" to="">
+            <img className="category-img" src="Images/tent.jpg" alt="" />
+            <span className="category-link-text">Camping</span>
+          </NavLink>
+        </section>
+        <section className="category">
+          <NavLink className="category-link" to="">
+            <img className="category-img" src="Images/shoes.webp" alt="" />
+            <span className="category-link-text">Footwear</span>
+          </NavLink>
+        </section>
+        <section className="category">
+          <NavLink className="category-link" to="">
+            <img className="category-img" src="Images/bag.jpg" alt="" />
+            <span className="category-link-text">Backpack</span>
+          </NavLink>
+        </section>
+        <section className="category">
+          <NavLink className="category-link" to="">
+            <img className="category-img" src="Images/light.jpg" alt="" />
+            <span className="category-link-text">Equipments</span>
+          </NavLink>
+        </section>
+      </section>
```

Comment:
You can extract the section-category into a component and then map over it instead of repeating it

Reply:
Done